trade-mapping-to-ipaffs/src/test/resources/chedd/chedd_ipaffs_minimum.json

```diff
@@ -0,0 +1,77 @@
+{
+  "id" : 0,
+  "version" : 0,
+  "type" : "CED",
+  "status" : "SUBMITTED",
+  "partOne" : {
+    "commodities" : {
+      "totalGrossWeight" : 1678.52,
+      "totalNetWeight" : 1678.52,
+      "temperature" : "Ambient",
+      "commodityComplement" : [ {
+        "commodityID" : "07099990",
+        "commodityDescription" : "Other",
+        "complementID" : 1,
+        "complementName" : "Aegle",
+        "speciesName" : "Aegle",
+        "speciesNomination" : "Aegle"
+      }, {
+        "commodityID" : "07099991",
+        "commodityDescription" : "Other",
+        "complementID" : 2,
+        "complementName" : "Aegle",
+        "speciesName" : "Aegle",
+        "speciesNomination" : "Aegle"
+      } ],
+      "complementParameterSet" : [ {
```

Comment:
Should be able to format this file now (because we're doing recursive comparison and not string comparison in the integration tests) so it doesn't have the weird `[ {` on the same line